duckduckgo/BrowserServicesKit
# New Privacy Dashboard as BSK submodule (Phase 3)

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/72649045549333/1203080778553137/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1358
macOS PR: https://github.com/more-duckduckgo-org/macos-browser/pull/797
What kind of version bump will this require?: Major

**Description**:
Part 1.  
It contains the following changes:
- move in the iOS Privacy Dashboard implementation as a separate module within BSK package
- extract required modules that the PrivacyDashboard depends on (ContentBlocking, UserScript)
The included code is **compatible with iOS** and not yet with the macOS implementation - no changes to platform (to follow in next PR).

**Steps to test this PR**:
1. Verify building iOS (no corresponding PR/branch, can be checked with a local copy)
2. Smoke test the Privacy Dashboard feature


**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
